content/uc-doc/contributors/contributing_doc.md

```diff
@@ -2,140 +2,101 @@
 
-    ## Section1
+#### Content
 
-    ### Subsection1
+Documentation content should be stored in `/content/uc-doc/`.
 
-    Foo.
+#### Table of Contents
 
-Incorrect:
+The table of contents is based on files and folders structure on `/content/uc-doc`.
+If you want to create a new level you must :
+- Create a new folder
+- Add an `index.md` with a list of subpages (Required to generate a clean navigation)
+- Create a new file `mypage.md`
+- Write your content inside the new file
 
-    ## Section1
+### Headings
 
-    Foo.
-
-    ## Section2
-
-    Bar.
+- The top section of each file must be capitalized using the following
+rule: capitalization of all words, except for articles, prepositions,
+conjunctions, and forms of to be.
+  ```
+  <!-- Correct -->
+  The Vitamins are in My Fresh California Raisins
+
+  <!-- Incorrect -->
+  The Vitamins Are In My Fresh California Raisins
+  ```
+- Must use `#` syntax to create a title
+- Must be H2 or lower. _(H1 titles are automatically generated by React)_
+  ```
+  ## My Title 1
+
+  ### My Subtitle 1.1
+
+  ## My Title 2
+  ```
+
+### Anchors and Headings
+  - **Default**: are automatically generated based on the title text
+  - **Custom**: words must be separated by dashes
+  ```
+  ## My Title {#title-custom-anchor}
+  ```
 
 ### Lists {#lists}
 
 Bullet lists:
 
-    * First item
-    * Second item
+    - First item
+    - Second item
+    - ...
 
 Autonumbered lists:
 
-    #. First item
-    #. Second item
-
-### Literal blocks {#literal-blocks}
-
-Use `::` on the same line as the line containing text when possible.
-
-The literal blocks must be indented with three spaces.
-
-Correct:
-
-    Bla bla bla::
-
-       apt-get update
+    1. First item
+    2. Second item
+    3. ...
 
-Incorrect:
+### Code blocks
 
-    Bla bla bla:
+Code blocks should have syntax highlight defined. Here's some values :
 
-    ::
-
-       apt-get update
-
-### Inline markup {#inline-markup}
-
-Use the following roles when applicable:
-
--   `:file:` for file, i.e.:
-
-        The :file:`/dev/null` file.
-
--   `:menuselection:` for interface menu:
-
-        The :menuselection:`Configuration --> Management --> Certificates` page.
-
--   `:guilabel:` for designating a specific GUI element:
-
-        The :guilabel:`Action` column.
+- ShellSession
+- Shell
+- YAML
+- ini
+- Python
+- Javascript
 
 ### Others {#others}
 
 -   There must be no warning nor error messages when building the
-    documentation with `make html`.
+    documentation with `yarn start`.
 -   There should be one and only one newline character at the end of
-    each file
+    each file.
 -   There should be no trailing whitespace at the end of lines
 -   Paragraphs must be wrapped and lines should be at most 100
-    characters long
-
-[^1]: `easy_install` can be found in the debian package
-    python-setuptools : `sudo apt-get install python-setuptools`
+    characters long
```

Comment:
lol these lines are not wrapped to 100
I know the spec mention *at most* but it make a bit weird to have many limit when not apparent reason

Reply:
I just keep it from the previous guidelines. Honestly, I prefer to not break paragraphs and use **word ward/line wrap** inside my editor.

If you and @sduthil are alright with this, I'll remove it from the guideline.

Reply:
I don't have opinion on this. With or without limit is fine with me, as long as we stick to the decision in all files.

Reply:
I prefer to keep the limit
* some editor do not wrap line with default config (ex: vscode) and it's harder to read 
* some editor do not handle very well really long long line (when the wrapped text don't fit in the screen) (ex: vim)
* using CLI tool (e.g. grep) is more suitable/readable when you don't have a long line
* navigate through very long line can be painful if you don't use mouse (moving word by word)
* diff are generally painful with long line (I guess you can say the same thing when wrapping line and adding one word change all line length)
* One long line is harder to point of something (I rarely say, hey you made a typo line 1 column 25093)

Reply:
Alright, I added new rules in prettier to respect the 100 characters long rule. We will add in another PR a Zull test that will be sure our Markdown are valid.